src/Mailjet/Client.php

```diff
@@ -183,4 +216,70 @@ public function setSecureProtocol($bIsSecured)
 
         return false;
     }
+    
+    // TODO : make the next code more readable
+    
+    /**
+     * Temporary set the variables generating the url
+     * @param array $options    contain temporary modifications for the client
+     * @param array $resource   may contain the version linked to the ressource
+     */
+    private function setOptions($options, $resource)
+    {
+        if (!empty($options['version']) && is_string($options['version'])) {
```

Comment:
Why not use [array_key_exists](http://php.net/manual/en/function.array-key-exists.php)?